projects/addon-commerce/components/input-card/input-card.component.ts

```diff
@@ -38,9 +38,9 @@ const icons = {
 };
 
 export function cardTextfieldControllerFactory(
-    directive: TuiTextfieldExampleTextDirective | null,
+    value: TuiTextfieldExampleTextDirective | null,
 ): TuiTextfieldExampleTextDirective {
-    directive = directive || new TuiTextfieldExampleTextDirective();
+    let directive = value || new TuiTextfieldExampleTextDirective();
     directive.exampleText = '0000 0000 0000 0000';
```

Comment:
Need an extra line around declaration. If there's no such rule in the new linter we need to add it.

Reply:
Not problem, in the next PR)